scripts/tests/test_signature.py

```diff
@@ -0,0 +1,82 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+from scripts.signature import bias, parse_args
+from pathlib import Path
+
+
+def write_to_file(file_path, data):
+    with open(file_path, "w") as f:
+        print(f"{data}", file=f)
+
+
+def read_from_file(file_path):
+    return Path(file_path).read_text()
+
+
+def create_bias(base_signatures, contender_signatures):
+    write_to_file("/tmp/base_signatures", base_signatures)
```

Comment:
is there a way to organize the functions in the file above in such a way that they can take the file contents as an argument, so that you don't need to write/read to files from unit tests?

Reply:
Organized it so that tests work with plain strings..